apps/src/blockly/addons/cdoFieldFlyout.js

```diff
@@ -0,0 +1,135 @@
+import GoogleBlockly from 'blockly/core';
+import CdoBlockFlyout from './cdoBlockFlyout';
+
+export default class CdoFieldFlyout extends GoogleBlockly.Field {
+  /**
+   * @param value The initial value of the field.
+   * @param {Object} opt_config A map of options used to configure the field.
+   *    Refer to the individual field's documentation for a list of properties
+   * this parameter supports.
+   */
+  constructor(value, opt_config) {
+    super(value);
+    this.configure_(opt_config);
```

Comment:
Could you update the description of the `opt_config` param so that reader knows that options could be passed in this customized field?

Reply:
Done!